cpp/tests/community/triangle_count_test.cpp

```diff
@@ -302,7 +320,10 @@ INSTANTIATE_TEST_SUITE_P(
   Tests_TriangleCount_Rmat,
   ::testing::Combine(
     // disable correctness checks for large graphs
-    ::testing::Values(TriangleCount_Usecase{0.1, false}, TriangleCount_Usecase{1.0, false}),
+    ::testing::Values(TriangleCount_Usecase{0.1, false, false},
+                      TriangleCount_Usecase{0.1, true, false},
+                      TriangleCount_Usecase{1.0, false, false},
+                      TriangleCount_Usecase{1.0, true, false}),
```

Comment:
For Larger graphs, I understand we want to have them all false.

Reply:
Yes, for benchmarking, we disable correctness check.